### DISCUSSION TITLE
How to include friend / protected friend modifier für vbnet?

### DISCUSSION THREAD

**Discussion by biopaule** [dotnet/docfx#7073] [Q&A]
Hello
I wonder if I can also include friend / protected friend modifier for vbnet ethods/properties. Currently only public methods/properties are processed.
Thanks and best regards

**Comment by JeremyCaney** [comment]
The equivalent of `Friend` in C# is `internal`. Issue #2561 addresses how to include `internal` classes. Essentially, you must disable [the default filter configuration](https://dotnet.github.io/docfx/tutorial/howto_filter_out_unwanted_apis_attributes.html#3-default-filter-configuration) by setting the `disableDefaultFilter` property in your `metadata` configuration to `true`. 

Doing so, however, will include a number of additional items that you may not want exposed. I imagine there’s a way to override _just_ that setting by establishing a custom `include` filter, but I’m not certain which of the default filter configuration rules is responsible for this.
